API/DTOs/Koreader/KoreaderProgressUpdateDto.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace API.DTOs.Koreader;
+
+public class KoreaderProgressUpdateDto
+{
+    public string Document { get; set; }
+    public DateTime Timestamp { get; set; }
```

Comment:
Is this in UTC? Added a comment would help.

Reply:
Yes UTC, I've added comments to both properties to add more detail